modules/osint/domain_reputation.py

```diff
@@ -149,6 +170,7 @@ def module_api(self):
 	if LISTS!=0:
 		presence = len(list(set(BLACKLIST))) / LISTS
 		OUTPUT['absence % on blacklist'] = 100 - (presence*100)
+	OUTPUT['Norton Safe Web Report:'] = RESULT
```

Comment:
wow. "Safe Web Report:" as a key? change it to safe_web_report.

Reply:
Eh...sorry, see how the output is printed? Wouldn't it look bad with the __ ?

Reply:
You can change the view in module_run but not in the json output